src/main/groovy/wooga/gradle/unity/tasks/RunCSScript.groovy

```diff
@@ -0,0 +1,82 @@
+package wooga.gradle.unity.tasks
+
+import org.gradle.api.file.RegularFile
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFile
+import wooga.gradle.unity.UnityTask
+
+import java.nio.file.Files
+
+class RunCSScript extends UnityTask {
+
+    private final RegularFileProperty sourceCsScript = objects.fileProperty()
+
+    @InputFile
+    RegularFileProperty getSourceCsScript() {

```

Comment:
Since these properties live inside the task, we can go wihtout the prefix as that is already inferred. `sourceScript`?